crates/core/tedge_actors/src/converter.rs

```diff
@@ -158,6 +177,50 @@ impl<C: Converter> ConvertingActor<C> {
     }
 }
 
+/// Connect and build a [ConvertingActor] wrapping a [Converter] with some `Config`.
+///
+/// The config can be any value that can be converted into the configuration expected by the message source.
+/// For instance, a converter that consumes MQTT messages needs to provide subscription topics.
+///
+/// ```no_run
+/// # use std::convert::Infallible;
+/// # use tedge_actors::Builder;
+/// # use tedge_actors::Converter;
+/// # use tedge_actors::ConvertingActor;
+/// # use tedge_actors::MessageSink;
+/// # use tedge_actors::MessageSource;
+/// # use tedge_actors::NoConfig;
+/// # use tedge_actors::ServiceProvider;
+/// # #[derive(Debug)]
+/// # struct MqttMessage;
+/// # #[derive(Clone)]
+/// # struct TopicFilter(&'static str);
+/// /// A Converter that translates MQTT messages
+/// struct MyConverter;
+///
+/// impl Converter for MyConverter {
+///     type Input = MqttMessage;
+///     type Output = MqttMessage;
+///     type Error = Infallible;
+///
+///     fn convert(&mut self, input: &Self::Input) -> Result<Vec<Self::Output>, Self::Error> {
+///         todo!()
```

Comment:
How about doing some simple conversion like duplicating the input message itself and returning that vector of duplicate messages?

Reply:
Why not.

My intent with this `todo!()` was to express the point is not here. This is about connecting the actor - not defining its behavior.